### ISSUE TITLE
Ideas for the Industrial Doors (Yes, plural)

### ISSUE THREAD

**Issue by Nova-Katt** [Rearth/Oritech#135]
### Describe the feature you'd like

1: So we have the really cool industrial door that looks great. However, the only door we get is one that can only be opened with redstone, like the iron doors and so on.
It would be nice if we **also** had a door that can be opened as usual, with the right click. No need for buttons or levers etc.

2: The other doors in the game can be combined from a 1x2 door to a 2x2 door by placing them side by side. The door from Oritech remains a single door, which also causes the gray door-frame to overlap eachother and glitch.
I would love if you could convert that into a 2x2 door as well when placed next to another door, so that the door-frame expands around both doors, with the swirly lock bolt ending up in the middle or something.

3: Trapdoors! both redstone locked and regular right click unlocked!

4: And something I really miss from Ad Astra.. the 3x3 doors. They had locked and unlocked 3x3 doors, they also had airlock doors, and blast doors, different designs on them, and some could be opened with right click, some had to be unlocked with redstone.

**Comment by Nova-Katt** [comment]
Oh in addition. There is a mod called Iron Door Key, which is a key that can open iron doors and it's likes with a right click, instead of buttons and levers.
You need to hold the key in your hand and click the door.

It defaults to the vanilla iron doors and similar, but via a tag in a datapack you can add more doors that it will work with. For instance the Obsidian Reinforced Door from Blockus, and so on.
But that doesn't work with the Industrial Door for some reason, so I guess it's made in a different way from the regular iron doors and trapdoors.

**Comment by ShrekTheGod42** [comment]
Yes being able to make the industrial doors 2x2 would be awesome

**Comment by kienerxyzzy** [comment]
at least make the borders connect

**Comment by Rearth** [comment]
Hi, thanks for the suggestions. I did some modelling in blockbench to try and get a good looking 2x2 and 2x3 door, but didnt get to anything I was really happy with. So I'll close this issue for now and maybe revisit it again in the future. If someone wants to help with the animated models that'd also be welcome of course.
